xarray/ufuncs.py

```diff
@@ -44,10 +44,12 @@ def __init__(self, name):
         self._name = name
 
     def __call__(self, *args, **kwargs):
-        _warnings.warn(
-            'xarray.ufuncs will be deprecated when xarray no longer supports '
-            'versions of numpy older than v1.13. Instead, use numpy ufuncs '
-            'directly.', PendingDeprecationWarning, stacklevel=2)
+        if self._name not in ['angle', 'iscomplex']:
+            _warnings.warn(
+                'xarray.ufuncs will be deprecated when xarray no longer '
+                'supports versions of numpy older than v1.13. Instead, use '
+                'numpy ufuncs directly.',
+                PendingDeprecationWarning, stacklevel=2)
```

Comment:
It might be better to say: "xarray.ufuncs will be deprecated when xarray no longer supports versions of numpy older than v1.17." We'll need `__array_function__` from 1.17 to override `angle` and `iscomplex`.

Or perhaps we should just drop the pending deprecation warning entirely given how far away this is (NumPy hasn't even released 1.16 yet, nevermind 1.17)

Reply:
Updated the warning message as you suggested.

I think independently of when `xarray.ufuncs` is going to be deprecated, it is good to let the user know that they can use numpy ufuncs now (for most functions except a few for which this warning is not raised)